specification/quantum/resource-manager/Microsoft.Quantum/preview/2022-04-05-preview/quantum.json

```diff
@@ -0,0 +1,1000 @@
+          "type": "string",
+          "description": "A description about this provider.",
+          "readOnly": true
+        },
+        "providerType": {
+          "type": "string",
+          "description": "Provider type.",
+          "readOnly": true
+        },
+        "company": {
+          "type": "string",
+          "description": "Company name.",
+          "readOnly": true
+        },
+        "defaultEndpoint": {
+          "type": "string",
+          "description": "Provider's default endpoint.",
+          "readOnly": true
+        },
+        "aad": {
+          "type": "object",
+          "description": "Azure Active Directory info.",
+          "properties": {
+            "applicationId": {
+              "type": "string",
+              "description": "Provider's application id.",
+              "readOnly": true
+            },
+            "tenantId": {
+              "type": "string",
+              "description": "Provider's tenant id.",
+              "readOnly": true
+            }
+          }
+        },
+        "managedApplication": {
+          "type": "object",
+          "description": "Provider's Managed-Application info",
+          "properties": {
+            "publisherId": {
+              "type": "string",
+              "description": "Provider's publisher id.",
+              "readOnly": true
+            },
+            "offerId": {
+              "type": "string",
+              "description": "Provider's offer id.",
+              "readOnly": true
+            }
+          }
+        },
+        "targets": {
+          "type": "array",
+          "description": "The list of targets available from this provider.",
+          "items": {
+            "$ref": "#/definitions/TargetDescription"
+          }
+        },
+        "skus": {
+          "type": "array",
+          "description": "The list of skus available from this provider.",
+          "items": {
+            "$ref": "#/definitions/SkuDescription"
+          }
+        },
+        "quotaDimensions": {
+          "type": "array",
+          "description": "The list of quota dimensions from the provider.",
+          "items": {
+            "$ref": "#/definitions/QuotaDimension"
+          }
+        },
+        "pricingDimensions": {
+          "type": "array",
+          "description": "The list of pricing dimensions from the provider.",
+          "items": {
+            "$ref": "#/definitions/PricingDimension"
+          }
+        }
+      }
+    },
+    "TargetDescription": {
+      "type": "object",
+      "description": "Information about a Target. A target is the component that can process a specific type of Job.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique target id."
+        },
+        "aliases": {
+          "type": "array",
+          "description": "The list of aliases for the target.",
+          "items": {
+            "type": "string"
+          }
```

Comment:
This is the only new change from previous version  - a new property is being added